splink/cluster_metrics.py

```diff
@@ -19,38 +20,40 @@ def _size_density_sql(
         sql string for computing cluster size and density
     """
 
-    # Get physical table names from Splink dataframes
-    edges_table = df_predict.physical_name
-    clusters_table = df_clustered.physical_name
-
-    input_col = InputColumn(_unique_id_col)
-    unique_id_col_l = input_col.name_l
+    # Get unique id columns from linker
+    uid_cols = self._settings_obj._unique_id_input_columns
+    # Create unique id for left-hand edges from unique (eg person) id and source dataset
+    uid_edges_l = _composite_unique_id_from_edges_sql(uid_cols, "l")
+    # Create unique id for clusters table
+    uid_clusters = _composite_unique_id_from_nodes_sql(uid_cols)
 
     sqls = []
+    # Count edges per node at or above a given match probability
     sql = f"""
         SELECT
-            {unique_id_col_l},
+            {uid_edges_l} AS unique_id_l,
```

Comment:
Is there a better name (alias) for this than `unique_id_l`?  `edge_group_id` perhaps? I think giving this a more semantically meaningful name would make the sql easy to read

Reply:
Good point. Have changed to `edge_group_id` and made some other variable names a bit more descriptive too